src/clr-angular/data/datagrid/datagrid.ts

```diff
@@ -215,4 +292,10 @@ export class ClrDatagrid implements AfterContentInit, AfterViewInit, OnDestroy {
     resize(): void {
         this.organizer.resize();
     }
+
+    @ViewChild("projectedDisplayColumns", {read: ViewContainerRef}) projectedDisplayColumns: ViewContainerRef;
+    @ViewChild("projectedCalculationColumns", {read: ViewContainerRef}) projectedCalculationColumns: ViewContainerRef;
+    @ViewChild("displayedRows", {read: ViewContainerRef}) displayedRows: ViewContainerRef;
+    @ViewChild("calculationRows", {read: ViewContainerRef}) calculationRows: ViewContainerRef;
+    public showDisplayTable = false;  // Init to calculate when a datagrid is created.
```

Comment:
This does not appear to be used anywhere.

Reply:
It is used in the displayMode subscription.  line #250/#259
I removed the initialization value since display mode service switched to a behavior subject.